ocamltest/ocamltest_stdlib.mli

```diff
@@ -62,4 +62,4 @@ module StringSet : sig
   val string_of_stringset : t -> string
 end
 
-module StringMap : Map.S with type key = string
+module StringMap = Misc.StringMap
```

Comment:
Why does this alias need to be re-exported?

Reply:
(Maybe `Ocamltest_stdlib` could just be `Misc.Stdlib`?)

Reply:
Done.